tests/acceptance/ape-config.yaml

```
@@ -5,8 +5,8 @@ plugins:
 
 dependencies:
   - name: nucypher-contracts
-    github: nucypher/nucypher-contracts
-    ref: main
+    github: cygnusv/nucypher-contracts
+    ref: states
```

Just so that we don't forget
```suggestion
    github: nucypher/nucypher-contracts
    ref: main
```